libs/features/mas/web-components/src/variants/ccd-suggested.js

```diff
@@ -112,6 +112,7 @@ export class CCDSuggested extends VariantLayout {
       font-size: var(--consonant-merch-card-body-xs-font-size);
       line-height: var(--consonant-merch-card-body-xs-line-height);
       min-width: fit-content;
+      white-space: pre-wrap;
```

Comment:
I recommend to set price slot as an inline-block or block instead of this.
![image](https://github.com/user-attachments/assets/c129a0a5-d067-4934-b246-4726f899b5d8)